packages/governance/src/types.js

```diff
@@ -450,6 +424,17 @@
  * @property {ParamValue} proposedValue
  */
 
+/**
+ * @typedef {Object} InvokeApiPosition
+ * @property {string} apiMethodName
+ * @property {any} methodArgs
```

Comment:
```suggestion
 * @property {unknown[]} methodArgs
```
or `any[]` if that makes a lot of errors. but we at least know that it's an array

Reply:
done